manifest.json

```diff
@@ -1,13 +1,11 @@
 {
   "name": "Assistive pricing",
   "description": "Handles assistive pricing, and allows for users to modify labels.",
+  "ubiquity:listeners": [ "issues.labeled", "issues.unlabeled", "label.edited", "issue_comment.created" ],
```

Comment:
Label edited is for editing labels not for adding and removing them on issues. Its not relevant. I assume the issue comment created is for the slash command. @gentlementlegen

Reply:
Yes, this covers all the functionalities.